.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v8.0/containers/dotnet
+{
+  "name": "dotnet",
+  "build": {
+    "dockerfile": "Dockerfile",
```

Comment:
Thanks for the feedback.

The main purpose of using `Dockerfile` is to keep some customization easy, nothing for now, so we could update to use `Dockerfile` when needed instead

Reply:
What customization are you anticipating? This version of the SDK installed isn't even really mandated by the base image, because the build.cmd script in the repository is going to redownload all required SDKs and Runtimes.

Reply:
(to be clear, this is great and I can't wait to merge it in)

Reply:
Maybe some system requirements like some libgdi+ for some test case, just some wondering, maybe not a real valid scene

Reply:
> This version of the SDK installed isn't even really mandated by the base image, because the build.cmd script in the repository is going to redownload all required SDKs and Runtimes.

Updated base image to `mcr.microsoft.com/devcontainers/base:debian` since we do not need to have dotnet installed. Use `debian` tag since we're using `debian` for the default dotnet docker image